libdnf5/base/log_event.cpp

```diff
@@ -191,9 +220,16 @@ std::string LogEvent::to_string(
         case GoalProblem::INSTALLED_IN_DIFFERENT_VERSION:
             if (action == GoalAction::REINSTALL) {
                 return ret.append(utils::sformat(
-                    _("Packages for argument '{}' installed and available, but in a different version => cannot "
-                      "reinstall"),
-                    *spec));
+                    _("Installed packages for argument '{}' are not available in repositories in the same version, "
```

Comment:
May I ask you to mark an order of arguments `{1}`. It will allow to change order for translators.

Reply:
Added.